tests/lib/rules/prefer-reflect.js

```diff
@@ -0,0 +1,272 @@
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.defineProperty({}, 'foo', { value: 1 })",
+            args: [2, { exceptions: "apply" }],
+            errors: [
+                {
+                    message: "Avoid using Object.defineProperty, instead use Reflect.defineProperty",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.getOwnPropertyDescriptor({}, 'foo')",
+            args: 2,
+            errors: [
+                {
+                    message: "Avoid using Object.getOwnPropertyDescriptor, instead use Reflect.getOwnPropertyDescriptor",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.getOwnPropertyDescriptor({}, 'foo')",
+            args: [2, { exceptions: "apply" }],
+            errors: [
+                {
+                    message: "Avoid using Object.getOwnPropertyDescriptor, instead use Reflect.getOwnPropertyDescriptor",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.getPrototypeOf({})",
+            args: 2,
+            errors: [
+                {
+                    message: "Avoid using Object.getPrototypeOf, instead use Reflect.getPrototypeOf",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.getPrototypeOf({})",
+            args: [2, { exceptions: "apply" }],
+            errors: [
+                {
+                    message: "Avoid using Object.getPrototypeOf, instead use Reflect.getPrototypeOf",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.setPrototypeOf({}, Object.prototype)",
+            args: 2,
+            errors: [
+                {
+                    message: "Avoid using Object.setPrototypeOf, instead use Reflect.setPrototypeOf",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.setPrototypeOf({}, Object.prototype)",
+            args: [2, { exceptions: "apply" }],
+            errors: [
+                {
+                    message: "Avoid using Object.setPrototypeOf, instead use Reflect.setPrototypeOf",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.isExtensible({})",
+            args: 2,
+            errors: [
+                {
+                    message: "Avoid using Object.isExtensible, instead use Reflect.isExtensible",
+                    type: "CallExpression"
+                }
+            ]
+        },
+        {
+            code: "Object.isExtensible({})",
+            args: [2, { exceptions: "apply" }],
```

Comment:
Can you use `options` instead of `args`?

`options: [{exceptions: "apply"}]`

You can leave off the initial 2, and any place with `args: 2` can just be removed.